app/ante/fee_checker_test.go

```diff
@@ -0,0 +1,49 @@
+package ante
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetTxPriority(t *testing.T) {
+	cases := []struct {
+		name        string
+		fee         sdk.Coins
+		gas         int64
+		expectedPri int64
+	}{
+		{
+			name:        "1 TIA fee large gas",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin("utia", 1_000_000)),
+			gas:         1000000,
+			expectedPri: 1000000,
+		},
+		{
+			name:        "1 utia fee small gas",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin("utia", 1)),
+			gas:         1,
+			expectedPri: 1000000,
+		},
+		{
+			name:        "2 utia fee small gas",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin("utia", 2)),
+			gas:         1,
+			expectedPri: 2000000,
+		},
+		{
+			name:        "1_000_000 TIA fee normal gas tx",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin("utia", 1_000_000_000_000)),
+			gas:         75000,
+			expectedPri: 13333333333333,
+		},
```

Comment:
All of these have a gasPrice of 1 or more. Can we add a test with a gas price (i.e. fee/gas or 0.1). Maybe gas is 1 million and fee is 1000utia

Reply:
good point added